docs/solutions/Hadoop/EL7/Cloudera/Development/scripts/bastion.sh

```diff
@@ -0,0 +1,194 @@
+#!/bin/bash
```

Comment:
A lot of these scripts appear to be duplicated between Development, prod, n-node. Can they be consolidated in a common scripts directory one level up?

Reply:
Yes they are common - I'm not sure the best way to consolidate this though and have the templates reference them properly?  The templates are built to stand alone - so pushing the scripts directory up to the root level here would require an end user to pull down all the profiles (rather than just the one they want).